sdk/pinocchio/Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = { workspace = true }
 readme = "./README.md"
 repository = { workspace = true }
 rust-version = { workspace = true }
+build = "build.rs" # keep 1.82.0 features out until program-tools allows us to move up from 1.79.0
```

Comment:
Let's bump the rust requirement here: https://github.com/anza-xyz/pinocchio/blob/main/Cargo.toml#L18

It should be ok to use `1.84.1` and then we don't need the build script.

Reply:
Good, bumped and removed build script etc. in ff76d903c98d58b6d0dee9491fe8cbf616a7e226